docs/user/benchmarking.md

```diff
@@ -0,0 +1,89 @@
+# Benchmarking
+
+This guide explains how to write and run garage benchmark scripts.
+
+## Prerequisites
+
+Firstly, you should have installed garage. If not, you can see instructions in the [latest documentation](https://garage.readthedocs.io/en/latest/user/installation.html) to install garage.
+
+Install `google-cloud-storage`:
+
+`pip install google-cloud-storage`
+
+## Install `garage-benchmarks`
+
+`cd garage/benchmarks && pip install -e .`
+
+## Write garage benchmark scripts
+
+**Define an experiment function**
+
+Use `@wrap_experiment` to define your experiment. `ctxt` is used for `@wrap_experiment`.
```

Comment:
Maybe this should link to the documentation of using wrap_experiment?
Perhaps https://garage.readthedocs.io/en/stable/user/experiments.html#running-experiments-on-gpu-cpu ?

Reply:
Agree. Will do.